### PR TITLE
[Static Validation] Ending parameter description with a dot

### PR DESCRIPTION
**Pull request by eriqua**
# Description

- Updating global tests to check if all parameter and output descriptions are ending with a dot and starting with a capital letter (.)
- Adding missing ending dots to parameter and output descriptions

| Pipeline |
| - |
| [KeyVault: Vaults](https://github.com/Azure/ResourceModules/actions/workflows/ms.keyvault.vaults.yml) |
| [Authorization: PolicyAssignments](https://github.com/Azure/ResourceModules/actions/workflows/ms.authorization.policyassignments.yml) (static validation OK) |
| [DesktopVirtualization: HostPools](https://github.com/Azure/ResourceModules/actions/workflows/ms.desktopvirtualization.hostpools.yml) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

# Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: arm/.global/global.module.tests.ps1 [hunk 1]
```diff
@@ -764,7 +764,7 @@ Describe 'Deployment template tests' -Tag Template {
             $outputs | Should -Contain 'resourceId'
         }
 
-        It "[<moduleFolderName>] parameters' description shoud start with a one word category followed by a dot, a space and the actual description text." -TestCases $deploymentFolderTestCases {
+        It "[<moduleFolderName>] parameters' description shoud start with a one word category followed by a dot, a space and the actual description text ending with a dot." -TestCases $deploymentFolderTestCases {
```
Comment (MrMCake) [inline arm/.global/global.module.tests.ps1:767]: Should the same be tested for the outputs? You updated those too

Comment (eriqua) [reply arm/.global/global.module.tests.ps1:767]: Was thinking about the same thing, but that would be an additional test. What do you think?

Comment (MrMCake) [reply arm/.global/global.module.tests.ps1:767]: I'd suggest to add one. More tests is not a bad thing - even if people that cloned the current repo may hate us for it. 
Test-wise it should be very simple as it's almost a 1:1 copy of the current test - just pointing to a different metadata block

Comment (eriqua) [reply arm/.global/global.module.tests.ps1]: done

**Review by MrMCake — APPROVED** [review]

**Pull request merged**